sdk/monitor/Azure.Monitor.Ingestion/perf/IngestionClientTest.cs

```diff
@@ -14,23 +16,42 @@ public class IngestionClientTest : PerfTest<IngestionClientTest.IngestionClientP
     {
         /* please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/perf/TemplateClientTest.cs to write perf test. */
 
+        protected static MonitorIngestionTestEnvironment TestEnvironment = new MonitorIngestionTestEnvironment();
```

Comment:
Could we possibly split this into two separate scenarios - BatchUpload (that calls convenience methods), SingleUpload that calls DPG method?